test/models/deploy_spec_test.rb

```diff
@@ -600,14 +592,45 @@ class DeploySpecTest < ActiveSupport::TestCase
       assert_match(/npm version/, @spec.review_checklist[0])
     end
 
+    test '#package_version returns the version number' do
+      file = Pathname.new('/tmp/fake_package.json')
+      file.write('{"version": "1.0.0-beta.1"}')
+
+      @spec.expects(:package_json).returns(file)
+      assert_equal '1.0.0-beta.1', @spec.package_version
+    end
+
+    test '#dist_tag returns "latest" if the version does not contains a pre-release tag' do
+      assert_equal 'latest', @spec.dist_tag('1.0.0')
+    end
+
+    test '#dist_tag returns "next" if the version contains a pre-release tag' do
+      assert_equal 'next', @spec.dist_tag('1.0.0-rc.1')
+    end
+
+    test '#npm? is false if npm package is private' do
```

Comment:
Please move this test back up to line 581.  Easier to read when private package tests are kept together.

Reply:
Good point, thanks. I regrouped them back together. The diff looks a bit weird but if you look at the whole file I think it's fine.